allennlp_models/modelcards/pair-classification-adversarial-binary-gender-bias-mitigated-roberta-snli.json

```diff
@@ -0,0 +1,70 @@
+{
+    "id": "pair-classification-adversarial-binary-gender-bias-mitigated-roberta-snli",
+    "registered_model_name": "adversarial_bias_mitigator",
+    "registered_predictor_name": "textual_entailment",
+    "display_name": "Adversarial Binary Gender Bias-Mitigated RoBERTa SNLI",
+    "task_id": "textual_entailment",
+    "model_details": {
+        "description": "This `Model` implements a basic text classifier and feedforward regression adversary with an adversarial bias mitigator wrapper. The text is embedded into a text field using a RoBERTa-large model. The resulting sequence is pooled using a cls_pooler `Seq2VecEncoder` and then passed to a linear classification layer, which projects into the label space. Subsequently, a `FeedForwardRegressionAdversary` attempts to recover the coefficient of the static text embedding in the binary gender bias subspace. While the adversary's gradients are computed normally, the classifier's gradients are computed such that updates to the classifier's parameters will not aid the adversary and will make it more difficult for the adversary to recover binary gender bias.",
```

Comment:
If this method really doesn't work that well, as you said in standup, then I think we should have a description of that somewhere.

Reply:
The method actually ended up working quite well.